File: Core/device/knowndevice.cpp
#include "knowndevice.h"

QLatin1String KnownDevice::PROPERTY_SERVICE_ID = QLatin1String("service_id");
QLatin1String KnownDevice::PROPERTY_NAME = QLatin1String("name");

KnownDevice::KnownDevice() : DeviceBase() {

}

KnownDevice::KnownDevice(QString id, QString serviceId, QString name, QObject *parent) : DeviceBase(id, serviceId, parent), m_name(name)
{

}

QString KnownDevice::name() {
    return m_name;
}

File: AndroidApp/OshCore/src/main/java/com/osh/device/KnownDevice.java
package com.osh.device;

import com.osh.SerializationSupport;

public class KnownDevice extends DeviceBase implements SerializationSupport {
	
	private String name;

	public KnownDevice(String id, String serviceId, String name) {
		super(id, serviceId);
		this.name = name;
	}

	public KnownDevice() {
		super();
	}

	public String getName() {
		return name;
	}

}
